which-key.el

```diff
@@ -525,24 +525,6 @@ it."
   :group 'which-key
   :type 'boolean)
 
-(defcustom which-key-enable-extended-define-key nil
```

Comment:
Looking at the code again, this variable could still toggle the use of menu item replacements (default to `t`) so that a user could toggle between showing command name vs replacement string on the fly. Thoughts on that? If that doesn't sound good I'll just do `make-obsolete-variable` as you suggest.